backend_py/primary/primary/routers/vfp/router.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from typing import List
+
+from fastapi import APIRouter, Depends, Query
+from primary.auth.auth_helper import AuthHelper
+from primary.services.sumo_access.vfp_access import VfpAccess
+from primary.services.sumo_access.vfp_types import VfpProdTable
+from primary.services.utils.authenticated_user import AuthenticatedUser
+from webviz_pkg.core_utils.perf_timer import PerfTimer
+
+from . import schemas
+
+LOGGER = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get("/vfp_table_names/")
+async def get_vfp_table_names(
+    # fmt:off
+    authenticated_user: AuthenticatedUser = Depends(AuthHelper.get_authenticated_user),
+    case_uuid: str = Query(description="Sumo case uuid"),
+    ensemble_name: str = Query(description="Ensemble name"),
+    realization: int = Query(description="Realization"),
+    # fmt:on
+) -> List[str]:
+    timer = PerfTimer()
```

Comment:
Consider using `ResponsePerfMetrics` instead of `PerfTimer`.
This will add the timings to the request and can then be accessed from the browser.